examples/MLIRSparseTensor/sparse-tensor-expand.mlir

```diff
@@ -0,0 +1,111 @@
+#CSR = #sparse_tensor.encoding<{
+  dimLevelType = [ "dense", "compressed" ]
+}>
+
+// Use linalg.matmul to multiply matrix
+func.func @matmul(%A: tensor<4x2xf64, #CSR>,
+                  %B: tensor<2x4xf64, #CSR>) -> tensor<4x4xf64, #CSR> {
+  %C = bufferization.alloc_tensor() : tensor<4x4xf64, #CSR>
+  %D = linalg.matmul
+    ins(%A, %B: tensor<4x2xf64, #CSR>, tensor<2x4xf64, #CSR>)
+    outs(%C: tensor<4x4xf64, #CSR>) -> tensor<4x4xf64, #CSR>
+  return %D: tensor<4x4xf64, #CSR>
+}
+
+// @matmul will be rewrited into code like @matmul_expand.
+// It use `sparse_tensor.expand` to get the innermost level access pattern of the given tensor.
+// This operation is useful to implement calculation kernel on the output kernel.
+// The `values` array contains values in the innermost level of the given tensor.
+// The `filled` array contains boolean value that indicate whether a coordinate has been filled at the level.
+// The `added` array and `count` array are used to store new level-coordinate when the current operating coordinate has never been filled.
+// After operation, the rewrite op will use `sparse_tensor.compress` to compress the access pattern into the given tensor.
```

Comment:
The "After operation" here seems ambiguous, can we use the statement like "After expanding", or "After above operations"?